libs/confman/src/detail/config_dbus_client.cpp

```diff
@@ -72,8 +85,16 @@ void config_dbus_client::initialize() {
           return { .value = this->value_call_(), .schema = this->schema_call_() };
         });
 
-    dbus_interface_->initialize();
+    // If the provided interface is created by this class we initialize it here,
+    // otherwise we assume it is taken care of elsewhere
+    if (!interface_name_.empty()) {
+      dbus_interface_->initialize();
+    }
```

Comment:
So you create a interface to share with multiple parties.
then one of the parties initalizes the interface?

Reply:
yes, similar to dbus connection